src/controllers/member.js

```diff
@@ -1,11 +1,67 @@
-// const updateMember = (req, res) => {
-//   console.log(req.params.id);
-// };
+const Member = require('../models/member');
 
-// const deleteMember = (req, res) => {
-//   console.log(req.params.id);
-// };
+const updateMember = (req, res) => {
+  const { id } = req.params;
+  const {
+    firsName,
+    lastName,
+    dni,
+    phone,
+    email,
+    password,
+    city,
+    birthDay,
+    postalCode,
+    isActive,
+    membership,
+  } = req.body;
 
-// module.exports = {
-//   updateMember, deleteMember,
-// };
+  Member.findByIdAndUpdate(
+    id,
+    {
+      firsName,
+      lastName,
+      dni,
+      phone,
+      email,
+      password,
+      city,
+      birthDay,
+      postalCode,
+      isActive,
+      membership,
+    },
+    { new: true },
+  )
+    .then((result) => {
+      if (!result) {
+        return res.status(404).json({
+          msg: `The member with id: ${id} was not found`,
+        });
+      }
+      return res.status(200).json(result);
+    })
+    .catch((error) => res.status(400).json(error));
+};
+
+const deleteMember = (req, res) => {
+  const { id } = req.params;
+  Member.findByIdAndDelete(id)
+    .then((result) => {
+      if (!result) {
+        return res.status(404).json({
+          msg: `The member with id: ${id} was not found`,
+        });
+      }
+      return res.status(204);
+    })
+    .catch((error) => res.status(400).json({
```

Comment:
The error occur in case of a database failure, so it would need to have a status of 500.

Reply:
Corrected!..